### DISCUSSION TITLE
nx doesn't fail on typescript errors

### DISCUSSION THREAD

**Discussion by amir-khoshbakht** [nrwl/nx#16665] [General]
TLDR: default empty nest project wont compile into .js when there are typescript errors ( ex: TS-2345 ERROR) but Nx compiles the typescript code even if there are errors in code. how ?

when I start a new nest project using :
```
npm i -g @nestjs/cli
nest new project-name
```
The generated project will not compile when there is an error in the typescript.
But when I create an integrated monorepo and add a nestjs app to it, the created app compiles even if there is an error.
How to set the project in such a way that if there is an error in the typescript, the compilation is not done (such as an empty Nest JS project)?

update:
It seems to be related.
 as nx docs says : 
_By default, Nx sets up your ESLint configs with performance in mind - we want your linting to run as fast as possible_
reference : [Using ESLint in Nx Workspaces](https://nx.dev/recipes/other/eslint)
